### PR TITLE
feat: add view column icons

### PR DESCRIPTION
**Pull request by feliciofilipe**
<div>
  Screenshot 2023-01-27 at 10 10 23
  Screenshot 2023-01-26 at 18 43 31
  Screenshot 2023-01-26 at 18 44 00
  Screenshot 2023-01-26 at 18 43 46
</div>

### CODE REVIEW

File: packages/ui-components/src/components/icon/icon.types.ts [hunk 1]
```diff
@@ -148,6 +148,8 @@ export enum EIconName {
 	Upgrade = 'kv-upgrade',
 	Upload = 'kv-upload',
 	Uptime = 'kv-uptime',
+	ViewColumn = 'kv-view-column',
+	ViewColumn2 = 'kv-view-column-2',
```
Comment (djcouto) [inline packages/ui-components/src/components/icon/icon.types.ts:152]: These names and not very explicit.

**Review by djcouto — CHANGES_REQUESTED** [review]
Can you please add a screenshot of the icons you are adding?

**Review by djcouto — APPROVED** [review]

File: packages/ui-components/src/components/icon/icon.types.ts [hunk 2]
```diff
@@ -130,6 +133,8 @@ export enum EIconName {
 	SortZa = 'kv-sort-za',
 	Square = 'kv-square',
 	Status = 'kv-status',
+	StarActive = 'kv-star-active',
+	StarInactive = 'kv-star-inactive',
```
Comment (ricardogoncalves89) [inline packages/ui-components/src/components/icon/icon.types.ts:137]: the icon is the same with different opacity right? that opacity can be added by css you don't need to have different icons

**Review by ricardogoncalves89 — CHANGES_REQUESTED** [review]

File: packages/ui-components/src/components/icon/icon.types.ts (hunk 2 above)
Comment (djcouto) [reply packages/ui-components/src/components/icon/icon.types.ts:137]: Yes, but is used as an icon. I mean, in every single place where you want to use an inactive start you must pass this opacity setting. That's why I choose to make it an icon.

Comment (ricardogoncalves89) [reply packages/ui-components/src/components/icon/icon.types.ts:137]: that way we should have an inactive state on all icons. If you really need this feature, I suggest adding a property to `kv-icon` to set the inactive state.

Comment (djcouto) [reply packages/ui-components/src/components/icon/icon.types.ts:137]: I see your point, but we cannot be sure that all inactive icons will all the same state.

In this case, I'll remove the inactive icon svg and style it as need 👍

**Review by ricardogoncalves89 — APPROVED** [review]

**Pull request merged**